src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,108 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
+import { FindMovieTypes } from './FindMovieTypes';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ moviesList, onSetMoviesList }) => {
```

Comment:
```suggestion
export const FindMovie = ({ moviesList, onSetMoviesList }) => {
```
moviesList не певен що пошуковику треба знати про всі фільми, він має знайти один, зібрати і віддати його на верх, а якщо такий фільм вже існує то просто не додавати його в список, але точно не взаємодіяти зі всім списком фільмів